ui/app/src/components/SingleFileUpload/SingleFileUpload.tsx

```diff
@@ -342,14 +355,35 @@ export function SingleFileUpload(props: SingleFileUploadProps) {
       </form>
       <div className="uppy-progress-bar" />
       <div className="uploaded-files">
-        <Typography variant="subtitle1" component="h2" className={classes.description}>
-          {description}
+        {error ? (
+          <Alert
+            icon={false}
+            severity="error"
+            action={
+              <Tooltip title={<FormattedMessage defaultMessage="Dismiss" />}>
+                <IconButton onClick={() => resetState()} size="small">
+                  <CloseIcon />
+                </IconButton>
+              </Tooltip>
```

Comment:
Can we have some sort of retry logic? If users wish to chose a different file, they can use the regular button. Make sure that clears the error too.
```suggestion
              <Tooltip title={<FormattedMessage defaultMessage="Retry" />}>
                <IconButton onClick={() => resetState()} size="small">
                  <ReplayRoundedIcon />
                </IconButton>
              </Tooltip>
```